nesc/whip6/platforms/boards/cc26xxbased/cherry-v4/TemperatureReaderOnOffSwitch.nc

```diff
@@ -0,0 +1,36 @@
+/*
+ * whip6: Warsaw High-performance IPv6.
+ *
+ * Copyright (c) 2012-2018 Przemyslaw Gumienny
+ * All rights reserved.
+ *
+ * This file is distributed under the terms in the attached LICENSE
+ * files.
+ */
+
+/**
+ * @author Przemyslaw Gumienny
+ */
+
+generic module TemperatureReaderOnOffSwitch() {
+    provides interface OnOffSwitch;
+    provides interface Init @exactlyonce();
+    uses interface IOPin as IOPin;
+}
+
+implementation {
+    command error_t Init.init() {
+        call IOPin.makeOutput();
```

Comment:
Are we guaranteed to have a certain HIGH/LOW state here?

Reply:
fixed